ui/css/failure-summary.css

```diff
@@ -21,3 +21,10 @@
   /* Override the selector complexity of: "ul.failure-summary-list li .btn-xs" */
   font-size: 11px !important;
 }
+
+.failure-summary-new-message {
+  padding: 2px 4px 0;
+  font-size: 12px;
+  background-color: #f7c194;
```

Comment:
This and the next line could use CSS variables (or a separate class for the coloring).